ActindoWorkFlow/Operation/01_ManageWorkflows.md

```diff
@@ -61,11 +65,11 @@ Create a transition to define the desired business process.
 
 #### Prerequisites
 
-The basic settings of the workflow have been defined and the workflow editor is displayed, see [Define the workflow basic settings](#define-the-workflow-basic-settings).
+The basic settings of the workflow have been defined, and the workflow editor is displayed, see [Define the workflow basic settings](#define-the-workflow-basic-settings).
 
 #### Procedure
 
-*Workflows > Workflows > Tab OVERVIEW > Button Add > Button CREATE*
+*Process Orchestration > Workflows > Tab OVERVIEW > Button Add > Button CREATE*
```

Comment:
Ich denke, dieser Pfad stimmt hier nicht. Nachdem man ein Workflow erstellt hat (start und endplace, wie oben beschrieben), muss man auf die Liste von Workflows gehen, dann auf die Version 1 (by default erstellt) klicken und dann editieren, oder? Ich frage mich, ob wir Create transition und Add transition nicht in einer Prozedur zusammenfassen können...